# Team Project Name

List of participants and affiliations:
- Daniel Paiva Agustinho, Baylor COllege of Medicine (Team Leader)
- Sushant Potdar, University of Arkansas (Writer)
- Alexis Quintero, National University of Colombia
- Andrew Miller, University of Utah
- John Arnn,  Utah Public Health Laboratory
- Rong Guo, University of Maryland, College Park
- Participant, Affiliation

## Project Goals
The analysis of the genomic sequence of clinical isolates of SARS-CoV-2 can provide insights into circulating strains in the human population. However, this approach may present certain limitations. Clinical isolates primarily represent severe cases that required hospitalization, potentially leading to a biased view of the virus's diversity. As a result, the genomic sequences obtained from clinical isolates may not fully capture the prevalence and variation of milder or asymptomatic infections, which constitute the majority of cases in the population.

In contrast, wastewater sequencing offers a more inclusive approach, providing a broader representation of circulating strains without bias towards severe cases. Wastewater contains genetic material shed by individuals with varying degrees of infection, including asymptomatic and mild cases, making it more reflective of the overall viral diversity in a community. However, wastewater testing has its challenges. The conditions within wastewater can lead to RNA degradation, which may result in the potential underestimation of minor alleles or variants. The indiscriminate representation of genetic material in wastewater also includes viral RNA from non-infectious particles, making it necessary to carefully interpret the results.

In this project, we aim to compare wastewater sampling with clinical isolates from specific temporal and geographical subsets. Our goal is to evaluate the representativeness of each approach concerning the total population of variants found circulating in these subsets. By assessing the genomic sequences obtained from both wastewater and clinical isolates, we can gain valuable insights into the diversity and prevalence of SARS-CoV-2 strains within each community. This comparative analysis will help us understand the strengths and limitations of each approach, making it possible to provide a more comprehensive view of the viral dynamics in distinct populations.

## Approach

## Results

## Future Work
